doc/api/https.md

```diff
@@ -131,7 +131,7 @@ const https = require('https');
 const fs = require('fs');
 
 const options = {
-  pfx: fs.readFileSync('test/fixtures/test_cert.pfx'),
+  pfx: fs.readFileSync('test/fixtures/keys/rsa_cert.pfx'),
```

Comment:
This is end-user documentation. I don't think it needs to be updated. It's sample code, not something that is supposed to run.

Reply:
Should I revert the change? I feel like something like `pfx: fs.readFileSync('path/to/key.pfx')` or something like that would be less confusing. That could be a separate PR though.

Reply:
Better to revert. And I agree, path/to/key would be better, and it should be a separate documentation PR, if you would like to make the change.

Reply:
I've reverted the change with 2a3809ac0bfae1e7bb621a036f37f74a48ac992e. Tell me if there's any problem with how I reverted it!